app/src/main/res/layout/emoji_tile.xml

```diff
@@ -19,5 +19,5 @@
         android:layout_height="wrap_content"
         android:layout_gravity="center_vertical"
         android:layout_marginLeft="8dp"
-        android:text="TextView" />
+        android:text="@string/text_view" />
```

Comment:
Exactly, That's why i'm asking to put a placeholder and maybe wrap these in comments that these are just placeholders! 

To eliminate the warnings and for the travis CI we have to put these in strings.xml